plugins/premium/comments.md

```diff
@@ -46,30 +46,42 @@ Follow this procedure to edit a comment.
 
 Follow this procedure to delete a comment. This option is not available for the first comment in a conversation.
 
-1. Click on this ![3 dots]({{site.baseurl}}/images/3dots.png) icon above the comments box to expand the menu.
+1. Click on this ![3 dots]({{site.baseurl}}/images/comments-3dots.png) icon above the comments box to expand the menu.
 1. Select **Delete** from the menu items.
 1. The following options appear in the comments sidebar:<br/>
-![delete comment]({{site.baseurl}}/images/delete.png)
+![delete comment]({{site.baseurl}}/images/comments-delete-comment.png)
 1. Click **Cancel** to save or **Delete** to remove the comment from the conversation.
 
 ### Delete conversation
 
 This option is only available for the first comment in a conversation. Once the comment is saved, follow this procedure to delete a conversation.
 
-1. Click on this ![3 dots]({{site.baseurl}}/images/3dots.png) icon above the comments box to expand the menu.
+1. Click on this ![3 dots]({{site.baseurl}}/images/comments-3dots.png) icon above the comments box to expand the menu.
 1. Select **Delete conversation** from the menu items.
 1. The following decision dialog box will appear:<br/>
-![delete conversation]({{site.baseurl}}/images/decision.png)
+![delete conversation]({{site.baseurl}}/images/comments-delete-conversation.png)
 1. Click **Cancel** to save or **Delete** to remove the conversation.
 
 **Result**: The conversation and all its subsequent comments will be deleted.
 
+### Resolve conversation
+
+This option is only available for the first comment in a conversation. Once a comment is saved, follow this procedure to resolve a conversation.
+
+1. Click on this ![3 dots]({{site.baseurl}}/images/comments-3dots.png) icon above the comments box to expand the menu.
```

Comment:
Looking at: https://www.tiny.cloud/docs/configure/contributing-docs/#talkingabouttheuserinterface
We would use 'select' per https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui
however this is out of scope for this PR. It would lead to replacing a lot of 'clicks'.
also, comments isn't supported on mobile (which may only be related to the sidebar size): https://www.tiny.cloud/docs/mobile/

Reply:
```suggestion
1. Click on the ellipsis ![(ellipsis - 3 horizontal dots)]({{site.baseurl}}/images/icons/more-drawer.svg) icon above the comment box to expand the menu.
```